third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -641,11 +641,45 @@ resource "google_compute_instance_group_manager" "my_igm" {
 ### `update_strategy` has been replaced by `update_policy`
 
 To allow much greater control over the updates happening to instance groups
-`update_strategy` has been replaced by `update_policy`. The previous
-functionality to determine if instance should be replaced or restarted can be
-achieved using `update_policy.minimal_action`. For more details see the
+`update_strategy` has been replaced by `update_policy`. The functionality controlled by `update_strategy` is now controlled by a combination of `update_policy.type` and `update_policy.minimal_action`. `update_strategy = NONE` can be achieved with `type = OPPORTUNISTIC`. The previous values of `RESTART` and `REPLACE` were both `PROACTIVE` types implicitly previously but can now be controlled explicitly.
+
+For more details see the
 [official guide](https://cloud.google.com/compute/docs/instance-groups/rolling-out-updates-to-managed-instance-groups).
 
+### Old Config
+
+```hcl
+resource "google_compute_instance_group_manager" "my_igm" {
+    name               = "my-igm"
+    zone               = "us-central1-c"
+    base_instance_name = "igm"
+
+    instance_template = "${google_compute_instance_template.my_tmpl.self_link}"
```

Comment:
nit: we're not super consistent throughout the guide, but we probably should use 0.12 syntax

Reply:
Looks like 0.11 syntax is rampant. Will send up another PR cleaning up all of the samples at once.